src/scripts.js

```diff
@@ -141,39 +145,51 @@ function openModal(matchingRecipe) {
           </article>
           <div class='modal-icons'>
             <div class='favorite-heart' id='favoriteHeart'>
-              <img src="https://img.icons8.com/pastel-glyph/64/000000/hearts--v1.png"/>
+              <img src='https://img.icons8.com/pastel-glyph/64/000000/hearts--v1.png' class='favorite-icon' id='favoriteIcon'/>
             </div>
             <div class='add-to-cook' id='addToCook'>
-              <img src="https://img.icons8.com/ios/50/000000/plus--v1.png"/>
+              <img src='https://img.icons8.com/ios/50/000000/plus--v1.png' class='add-to-cook-icon' id='addToCookIcon'/>
             </div>
           </div>
         </div>`
+  openModal();
+}
+
+function openModal() {
   recipeModal.style.display = 'flex';
 }
 
 function closeModal(event) {
-  //used what you already set up to make the modal close work so I 
-  //can keep working 
+  //used what you already set up to make the modal close work so I
+  //can keep working
   if (event.target.id === 'closeModal') {
     recipeModal.innerHTML = '';
     recipeModal.style.display = 'none';
   }
 }
 
-function checkFavorites(event) {
-  const favorites = recipeCollection.recipes.forEach(recipe => {
-    if (event.target.id === recipe.id && !currentUser.favoriteRecipes.includes(recipe.id)) {
-      currentUser.favoriteRecipes.push(recipe) 
-    }
-  })
-
-}
+function addToFavoritesList(event) {
+  const clickedRecipe = parseInt(event.target.closest('article').id);
+  activate(favoriteRecipe);
```

Comment:
instead of passing this favoriteRecipe, we should pass it the event.target because activate and deactivate take an element, and the event.target is the specifically clicked heart html element.